src/app/collection/pages/collection-502/collection-502.component.ts

```diff
@@ -19,17 +20,41 @@ export class Collection502Component implements OnInit {
     limit: 5,
     collection: '502_project'
   };
+  currentTheme = 'light';
 
   constructor(
     private navbarService: NavbarService,
-    private learningObjectService: LearningObjectService
+    private learningObjectService: LearningObjectService,
+    private collectionService: CollectionService
   ) { }
 
   async ngOnInit() {
 
     this.navbarService.show();
     await this.fetchLearningObjects(this.query);
 
+    const toggleSwitch = document.querySelector('.theme-switch input[type="checkbox"]');
+
+    const switchTheme = (e: Event) => {
+      if ((e.target as HTMLInputElement).checked) {
+        this.collectionService.changeStatus502(true);
+      } else {
+        this.collectionService.changeStatus502(false);
+      }
```

Comment:
```suggestion
      this.collectionService.changeStatus502((e.target as HTMLInputElement).checked);
```
If `.checked` is a boolean, you can just pass it into the function instead of using the hardcoded true and false